apps/files/lib/Service/UserConfig.php

```
@@ -54,6 +54,12 @@ class UserConfig {
 			'default' => true,
 			'allowed' => [true, false],
 		],
+		[
+			// Whether to show the mime column or not
+			'key' => 'show_mime_column',
+			'default' => true,
```

Can we / should we default to false?

---

(especially as in many cases the value will be very technical)

---

Up to @marcoambrosini I'd say :)
Considering this is hidden on small screens, I wonder if that would be such a bad thing to have it enabled by default :thinking:

---

I would avoid showing it by default, at least until we have a way for users to hide columns that they don't need

---

Oh nice! I'm still leaning towards hiding tbh :)

---

Ok for me! :)

---

```suggestion
			'default' => false,
```